ts/private/ts_project.bzl

```diff
@@ -123,7 +127,7 @@ def _ts_project_impl(ctx):
             ctx.outputs.buildinfo_out.short_path,
         ])
         outputs.append(ctx.outputs.buildinfo_out)
-    runtime_outputs = json_outs + js_outs + map_outs
+    sources = json_outs + js_outs + map_outs
```

Comment:
this is a confusing rename

Reply:
meant to align with what is produced in JsInfo. `output_sources` clearer?

Reply:
updated to `output_sources`